docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster-slim
+
+RUN set -ex \
+    && apt-get update \
+    && apt-get install -y \
+	build-essential \
+	python2.7-dev \
+	libffi-dev \
+	sqlite3 \
+	python-pip \
+	libssl-dev \
+	python-virtualenv \
+	libxslt1-dev
+
+# Install dependencies
+RUN pip install https://github.com/matrix-org/sydent/tarball/master \
```

Comment:
One usually builds from the checked out source, rather than just always fetching master.